contrib/deploy-aap/verify.sh

```diff
@@ -0,0 +1,17 @@
+#!/usr/bin/env bash
+
+set -o pipefail
+set -o nounset
+set -m
+
+# variables
+# #########
+# uncomment it, change it or get it from gh-env vars (default behaviour: get from gh-env)
+# export KUBECONFIG=/root/admin.kubeconfig
+
+# Load common vars
+source ${WORKDIR}/shared-utils/common.sh
+
+echo "development"
```

Comment:
Just to make it clear, the idea of the contrib framework is to have an interface with 2 functions:
- Deploy.sh -> deploy resources for a specific resource/operator/item
- Verify.sh  -> verify if there's present to skip this task. The verification should be a functional test to know if the resources are ready and running.

So, said that, in this part an example of verify could be:

[Verify.sh](https://github.com/rh-ecosystem-edge/ztp-pipeline-relocatable/blob/33f65288208908623e96fd82269bd381c0ba6560/contrib/gpu-operator/verify.sh)